src/data/services/tests/EnterpriseAccessApiService.test.js

```diff
@@ -152,4 +152,15 @@ describe('EnterpriseAccessApiService', () => {
       `${enterpriseAccessBaseUrl}/api/v1/subsidy-access-policies/${mockSubsidyAccessPolicyUUID}/`,
     );
   });
+
+  test('allocateContentAssignments calls enterprise-access allocate POST API to create assignments', () => {
+    const payload = {
+      learner_emails: ['edx@example.com'],
```

Comment:
Nit: is it worth having a test payload that matches what the endpoint actually requires (i.e. a `content_key` and `content_price_cents`)?

Reply:
Updated!